src/components/Calculator.css

```diff
@@ -0,0 +1,56 @@
+.calculator {
+  width: 300px;
+  margin: 0 auto;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  text-align: center;
+}
+
+.display {
+  margin-bottom: 10px;
+}
+
+input[type="text"] {
+  width: 92%;
+  padding: 10px;
+  font-size: 24px;
+  color: #fff;
+  margin-bottom: -10px;
+  text-align: right;
+  background-color: rgb(161, 161, 161);
+}
+
+.buttons {
+  display: grid;
+  gap: 1px;
+}
+
+.row {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 1px;
+}
+
+button {
+  padding: 10px;
```

Comment:
- [optional] Great job making the changes. Please, i advise you add your calculator button sizes so it can be more visible. This will be more user-friendly. Nice work.